homeassistant/components/cloud/__init__.py

```diff
@@ -81,6 +83,25 @@
 }, extra=vol.ALLOW_EXTRA)
 
 
+@bind_hass
+@callback
+async def async_is_logged_in(hass):
+    """Test if user is logged in."""
+    return hass.data[DOMAIN].is_logged_in
+
+
+@bind_hass
+async def async_create_cloudhook(hass, webhook_id):
```

Comment:
Calling this when `cloud` is not loaded throws a `KeyError`. Should we add something like:

```python
if DOMAIN not in hass.data:
        return None
```

to the create, delete and `async_is_logged_in` calls? (For `async_is_logged_in` we should return `False` obviously)

Reply:
Also, should we automatically call `async_is_logged_in` in the create and delete functions and also return `None` if user is not logged in?

Reply:
`is_logged_in` should be false, the others should raise